pio/CMakeLists.txt

```diff
@@ -31,38 +30,40 @@ endif()
 
 # Netcdf is required
 
+SET (CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../pio2/cmake" ${CMAKE_MODULE_PATH})
```

Comment:
Once this PR goes in I can create an issue to cleanup CMakeLists.txt in pio1 (and make sure that ESMCI/cime works with this version of pio1)